hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/opt/physical/visitor/RexToExpression.java

```diff
@@ -282,6 +283,11 @@ private RexToExpression() {
 
                 // Math.
 
+                if (function == HazelcastSqlOperatorTable.ATAN2) {
```

Comment:
Let's keep the style consistent - integrate with `else-if`? Also, maybe move it next to `ATAN`?